contracts/NXMToken.sol

```diff
@@ -0,0 +1,336 @@
+/* Copyright (C) 2017 NexusMutual.io
+
+  This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/ */
+
+pragma solidity 0.4.24;
+
+import "./imports/openzeppelin-solidity/token/ERC20/IERC20.sol";
+import "./imports/openzeppelin-solidity/math/SafeMaths.sol";
+
+
+contract NXMToken is IERC20 {
+    using SafeMaths for uint256;
+
+    event WhiteListed(address indexed member);
+
+    event BlackListed(address indexed member);
+
+    mapping (address => uint256) private _balances;
+
+    mapping (address => mapping (address => uint256)) private _allowed;
+
+    mapping (address => bool) private canNotOperate;
```

Comment:
I think we can remove this mapping.

Reply:
okay, i will implement changes accordingly.